projects/packages/identity-crisis/src/class-ui.php

```diff
@@ -161,4 +165,27 @@ function ( $c1, $c2 ) {
 		return static::$consumers;
 	}
 
+	/**
+	 * Get the site platform.
+	 *
+	 * @return string
+	 */
+	private static function get_platform() {
+		$host = new Host();
+
+		if ( $host->is_woa_site() ) {
+			return 'woa';
+		}
+
+		if ( $host->is_vip_site() ) {
+			return 'vip';
+		}
+
+		if ( $host->is_newspack_site() ) {
+			return 'newspack';
+		}
+
+		return 'standalone';
```

Comment:
Do we use `standalone` to refer to self-hosted sites elsewhere?

Reply:
Perhaps `self-hosted` as we use in our internal health metrics

Reply:
Thanks @dereksmart!